File: mysql/connect.php
<?php
# connect.php: connect to the MySQL server

try {
    $dsn = "mysql:host=localhost;dbname=cookbook";
    $dbh = new PDO($dsn, "user", "password");
    print("Connected\n");
} catch (PDOException $e) {
    die("Cannot connect to server\n");
}
$dbh = NULL;
print("Disconnected\n");
?>

File: Backend/mysql/Connect.java
// Connect.java: connect to the MySQL server


import java.lang.Exception;
import java.lang.System;
import java.sql.*;

public class Connect {
    public static void main(String[] args) {
        Connection conn = null;
        String url = "jdbc:mysql://localhost/cookbook";
        String userName = "user";
        String password = "password";

        try {
            Class.forName("com.mysql.jdbc.Driver").newInstance();
            conn = DriverManager.getConnection(url, userName, password);
            System.out.println("Connected");
        } catch(Exception e){
            System.err.println("Cannot connect to server");
            System.exit(1);
            e.printStackTrace();
        }
        if (conn != null) {
            try {
                conn.close();
                System.out.println("Disconnected");
            } catch (Exception e) {
                // ignore close errors
            }
        }
    }
}
